ONSdigital/ras-frontstage
# Adding Pipfile.lock d31989

# What and why?

Pipfile.lock d31989 was missing from the merge to main for release 2.4.85.

While this didn't stop that release being built, it did mean PRs opened from main after that merge are broken as the pipenv doesn't detect the Pipfile needs re-generating.

# How to test?

Ensure the GHA build passes and also doesn't need to re-create the Pipfile.lock while installing the virtualenv.

As per this PR's GHA build https://github.com/ONSdigital/ras-frontstage/actions/runs/6850260177/job/18624077379#step:5:74
